lib/objects/reviewAwardReasonObject.js

```diff
@@ -73,8 +73,8 @@ function formatReasonForChange(reasonForChange) {
   case 'COEG INVESTIGATION C.O. POSITION - BP CALCULATION ONLY':
     reason = 'COEG investigation contracted out position - basic pension calculation only';
     break;
-  case 'RP BENEFIT CALCULATION MADE - FEDBACK DETAILS NOT RECEIVED':
-    reason = 'Retirement pension benefit calculation made - fedback details not received';
+  case 'RP BENEFIT CALCULATION MADE - FEEDBACK DETAILS NOT RECEIVED':
```

Comment:
Revert this, please. Will investigate if this is a spelling mistake. We are translating an API response here

Reply:
done